express-post-json/index.js

```diff
@@ -14,15 +14,15 @@ app.get('/api/grades', (req, res) => {
     })
   }
   res.json(gradesArray);
-})
+});
 
-app.use(express.json())
+app.use(express.json());
 
 app.post('/api/grades', (req, res) => {
   grades[nextId] = req.body;
   grades[nextId].id = nextId;
   res.status(201).send(grades[nextId]);
   nextId++;
-})
+});
```

Comment:
I'd verify that the request body contains the `name`, `course`, and `grade` properties. If any of those are missing, don't create the grade in `grades`, and send a `400 Bad Request` response instead.